presto-main/src/main/java/com/facebook/presto/server/BasicQueryStats.java

```diff
@@ -72,7 +78,11 @@ public BasicQueryStats(
             Duration totalCpuTime,
             boolean fullyBlocked,
             Set<BlockedReason> blockedReasons,
-            OptionalDouble progressPercentage)
+            OptionalDouble progressPercentage,
```

Comment:
This doesn't need to go in basicquerystats (it's only used for the list view on the UI) - revert